packages/aws-cdk-lib/custom-resources/test/custom-resource-config/custom-resource-config.test.ts

```diff
@@ -200,3 +200,54 @@ test('addLogRetentionLifetime modifies the retention period of the custom resour
     RetentionInDays: customResourceLogRetention,
   });
 });
+
+describe('when custom resource logGroup is default set to Retain', () => {
+  test('addRemovalPolicy modifies custom resource logGroup to Delete', () => {
+    // GIVEN
+    const customResourceRemovalPolicy = cdk.RemovalPolicy.DESTROY;
+    const app = new cdk.App();
+    const stack = new cdk.Stack(app);
+    const websiteBucket = new s3.Bucket(stack, 'WebsiteBucket', {});
+    new s3deploy.BucketDeployment(stack, 'BucketDeployment', {
+      sources: [s3deploy.Source.jsonData('file.json', { a: 'b' })],
+      destinationBucket: websiteBucket,
+      logGroup: new logs.LogGroup(stack, 'LogGroup', {}),
```

Comment:
What does this default to? This test isn't (and shouldn't be) testing that the default value is set to something, but since I don't know off-hand what this value defaults to, it would be very helpful to explicitly pass the value.